### PR TITLE
Testing forms

### PR DESCRIPTION
**Pull request by christina-cook**
## *Purpose:*

- [ ] Adds a new feature
- [x] Improves an existing feature
- [ ] Fixes a bug
- [x] Cleans up (logs, tests, etc.)

## *Description: What does this PR do?*

Tested initial functionality of login and signup forms.

## *What are the relevant tickets?*
 
Closes _

## Checklist:
- [x] I have performed a self-review of my own code
- [x] I have used tests to prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my change

## *Notes:*

We still need to test error messages (I didn't have the most recent branch pulled down for those) and also test an actual user signing up/logging in (with intercept). We will also need to test Google, Facebook, and GitHub signups once that functionality is working.

### CODE REVIEW

File: cypress/integration/login_spec.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+      .get('.form-label').should('have.length', '2')
+  })
+
+  it('Should be able to enter an email address and password into the form', () => {
+    cy.get('input[type=email]').type('test@gmail.com')
+      .get('input[type=password]').type('password')
+  })
+
+  it('Should see an error message if the email and password don\'t match', () => {
+    // test alert and error message
```
Comment (srslie) [inline cypress/integration/login_spec.js:23]: Great, once we get that alert that's on main in this those are set up :)

File: cypress/integration/login_spec.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+  // test Google login functionality
+
+  // test Facebook login functionality
+
+  // test GitHub login functionality
+
+  it('Should redirect user to the singup page if they don\'t have an account', () => {
+    cy.get('.form-footer').should('contain', 'Need to create an account? Sign Up')
+      .get('.signup-link').click().url().should('include', '/signup')
```
Comment (srslie) [inline cypress/integration/login_spec.js:40]: Woah! I didn't know you could chain a click().url(), for whatever reason I always do another cy.url()... neat

**Review by srslie — COMMENTED** [review]
Thanks for getting those up!

File: cypress/integration/login_spec.js (hunk 2 above)
Comment (christina-cook) [reply cypress/integration/login_spec.js:40]: I just tried it and it worked!

**Pull request merged**